addon/templates/components/polaris-card.hbs

```diff
@@ -60,3 +60,30 @@
     {{/if}}
   {{/if}}
 {{/with}}
+
+{{#if (or primaryFooterAction secondaryFooterAction)}}
+  <div class="Polaris-Card__Footer">
+    {{#polaris-button-group}}
+
+      {{#if primaryFooterAction}}
+        {{polaris-button
+          dataTestId="primaryFooterAction"
+          primary=true
+          text=primaryFooterAction.text
```

Comment:
can we drop in here and below `loading` property too? likely we're gonna use that